Remotion/Core/Core/Context/CallContextStorageProvider.obsolete.cs

```diff
@@ -15,28 +15,52 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
```

Comment:
Forgot: please update the filename (drop `.obsolete`). I only use the `.obsolete` suffix if the file contains only dead APIs without logic, not code that's still in use, even if it's obsolete. It's an indicator that the file can eventually be deleted.

- [ ] Rename